wayang-api/wayang-api-scala-java/src/main/scala/org/apache/wayang/api/JavaPlanBuilder.scala

```diff
@@ -112,6 +123,7 @@ class JavaPlanBuilder(wayangCtx: WayangContext, jobName: String) {
     */
   def withUdfJar(path: String) = {
     this.planBuilder withUdfJars path
+    System.out.println("**MK** ---MARKER--- in JavaPlanBuilder" )
```

Comment:
Please change this into a more understandable error message.

Reply:
Done.